Mathlib/Data/Finset/Powerset.lean

```diff
@@ -218,14 +218,21 @@ theorem card_powersetCard (n : ℕ) (s : Finset α) :
 #align finset.card_powerset_len Finset.card_powersetCard
 
 @[simp]
-theorem powersetCard_zero (s : Finset α) : Finset.powersetCard 0 s = {∅} := by
+theorem powersetCard_zero (s : Finset α) : s.powersetCard 0 = {∅} := by
   ext; rw [mem_powersetCard, mem_singleton, card_eq_zero]
   refine'
     ⟨fun h => h.2, fun h => by
       rw [h]
       exact ⟨empty_subset s, rfl⟩⟩
 #align finset.powerset_len_zero Finset.powersetCard_zero
 
+theorem powersetCard_one (s : Finset α) :
+    s.powersetCard 1 = s.map ⟨_, Finset.singleton_injective⟩ := by
+  ext; rw [mem_powersetCard, card_eq_one, mem_map, ← exists_and_left, exists_congr]
+  intro a
+  rw [eq_comm, ← s.singleton_subset_iff]
+  apply and_congr_left; rintro rfl; rfl
```

Comment:
It's not straightforward to use the Finset result to prove this, because there are two layers' difference: `Multiset (Multiset _)` vs `Finset (Finset _)`. One could use  [Finset.map_val_val_powersetCard](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Finset/Powerset.html#Finset.map_val_val_powersetCard) but it's at the end of the file.

Reply:
I think something along these lines would be good as a comment in the source code.

Reply:
I've moved [Finset.map_val_val_powersetCard](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Finset/Powerset.html#Finset.map_val_val_powersetCard) up and golfed the proof using the Multiset version.